src/nfa/operations.cc

```diff
@@ -597,6 +622,7 @@ Mata::Parser::ParsedSection Mata::Nfa::serialize(
     return {};
 }
 
+//method?
```

Comment:
Called `has_empty_lang()`?

Reply:
Don't have to say lang, as in the case of inclusion etc. I would just say empty(), or is_empty().

Reply:
We could, probably. I am not sure if that would cause more confusion, though. `empty()` is used in C++ to talk about the structural qualities of a container, not necessarily about its semantic impact.

Reply:
empty automaton and automaton with empty language are two different things to me

Reply:
I see, but is it really confusing? What is empty automaton then, and does it make sense to talk about it and give it a name, a function that tests it? 

I still like the idea that set operations and tests applied on automata always talk about the language, that would not be confusing if done consistently, no?
That other empty automaton could have some other more obscure name maybe, if it is needed. ?

Reply:
> I still like the idea that set operations and tests applied on automata always talk about the language, that would not be confusing if done consistently, no?

I like this idea. It is not what we do now, but if done consistently for operations on Nfa class (not Delta and its classes, mind you), it could work nicely. It would require some getting used to, though. Now, it is explicitly clear that we are talking about languages of automata with functions using `_lang_` somewhere in their names.